activerecord/lib/active_record/scoping/named.rb

```diff
@@ -27,6 +27,15 @@ def all
           scope = current_scope
 
           if scope
+            if scope._deprecated_scope?
+              ActiveSupport::Deprecation.warn(<<~MSG.squish)
+                Using class level querying methods in this context,
+                which the receiver scope regarded as leaked,
+                is deprecated and will not be leaked in Rails 6.1.
+                Use `#{name}.unscoped` to avoid the leaking scope.
```

Comment:
I took a swing at this message:

```Class level methods will no longer inherit scoping from #{scope._deprecated_scope_source} in Rails 6.1. To continue using the scoped relation, pass it into the block directly. To instead access the full set of models, as Rails 6.1 will, use `#{name}.unscoped`.```

Reply:
What we'd like to see at `#{scope._deprecated_scope_source}`?

Reply:
It is hard to show dynamic value without `#{name}` in this context.
It could be to be ```Class level methods will no longer inherit scoping from the scoped relation in Rails 6.1. To continue using the scoped relation, pass it into the block directly. To instead access the full set of models, as Rails 6.1 will, use `#{name}.unscoped`.```

Reply:
ah I misunderstand that. If we'd like "To continue using the scoped relation", we can use `#{name}.all` to drop the deprecated flag.

Reply:
ah couldn't use klass.all in this context for the purpose...

Reply:
I was thinking that `_deprecated_scope_source` would be something like ``` `create` ```

Reply:
Okey, addressed at https://github.com/rails/rails/compare/f86693cd04f19817028d189caec7430ca7df836d..18f0ca7bfdcb103323dc6f951301cefbf5ebc28a.